python/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ endif(CCACHE_FOUND)
 ############################################################
 
 # compiler flags that are common across debug/release builds
-set(CXX_COMMON_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall")
```

Comment:
Factored out common cmake code between C++/Python libraries, so you can set `CMAKE_CXX_FLAGS` to modify externally